### ISSUE TITLE
Authorize.net error with KP - Expiration & Card Security Code required fields [PI-597]

### ISSUE THREAD

**Issue by kristibarbosky** [krokedil/klarna-payments-for-woocommerce#120]
KP test order is successful when credit card details are entered

When credit card details are removed from the checkout, error is returned: 

Expiration (MM/YY)  is a required field.
Card Security Code  is a required field.

![image](https://user-images.githubusercontent.com/15654455/51403042-4230b280-1b1d-11e9-8093-6e3b34d21402.png)

Test environment login details will be provided via email.

In case relevant, please also see: https://github.com/krokedil/klarna-payments-for-woocommerce/issues/21

**Comment by MichaelBengtsson** [comment]
This seems to be a bug with our plugin that stems from the fix we did to block the iframe from opening up if not all required fields are filled in when submitting the order. From the commit fc89056cc82c13e77e334782ef6985a98ecc6f50 

I will take a look at this tomorrow and see if we can exclude the fields in the payment options part of the form. However we cant just limit this to inside of the adress fields, since this fix was implemented to support things like checkboxes outside of the normal form like the germanized plugin adds.
